bats/tests/helpers/paths.bash

```diff
@@ -48,6 +48,15 @@ set_path_resources() {
     if [ ! -d "$PATH_RESOURCES" ]; then
         fatal "App resource directory '$PATH_RESOURCES' does not exist"
     fi
+
+    # Ensure we're using bundled executables for credential helpers; normally we
+    # invoke everything else with the full path.
+    export PATH="$PATH_RESOURCES/$OS/bin:$PATH"
+    if is_windows; then
+        # Use Linux utilities (like jq) on WSL.
+        export PATH="$PATH_BATS_ROOT/bin/${OS/windows/linux}:$PATH"
```

Comment:
This is not correct. The `PATH` needs to be modified on all platforms.

The comment is only explaining the substitution `${OS/windows/linux}` instead of just using `$OS`.